wireless/bluetooth/bt_smp.c

```diff
@@ -71,6 +71,15 @@
 
 /* SMP channel specific context */
 
+enum pairing_method {
+    PAIRING_METHOD_JUST_WORKS,
+    PAIRING_METHOD_PASSKEY_DISPLAY, // Local displays, remote inputs
+    PAIRING_METHOD_PASSKEY_INPUT,   // Local inputs, remote displays
```

Comment:
Please fix code style issues, there are many C++ "//" comments and even "{" starting at end of the line